lib/components/molecules/StickyHeader/tests/StickyHeader.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { StickyHeaderVanilla } from '../StickyHeader';
+
+describe('Test render <StickyHeader /> Component', () => {
```

Comment:
If someone removes the scroll and sticky behaviour from the components code will any test case report it?